src/main/scala/org/broadinstitute/hail/expr/package.scala

```diff
@@ -23,4 +22,6 @@ package object expr extends HailRepFunctions {
   implicit def toRichParser[T](parser: Parser.Parser[T]): RichParser[T] = new RichParser(parser)
 
   type CPS[T] = (T => Unit) => Unit
+  type CodeCPS[T] = (Code[T] => Code[Unit]) => Code[Unit]
```

Comment:
I think this might actually be dead code. I created it on my way to `CMCodeCPS`.